src/main/java/edu/colorado/cires/cmg/tracklinegen/BaseRowListener.java

```diff
@@ -49,11 +50,12 @@ public class BaseRowListener<T extends DataRow> implements RowListener<T> {
    * @param maxAllowedSimplifiedPoints
    * @param geometryFactory
    * @param geoJsonPrecision
-   * @deprecated Use {@link BaseRowListener#BaseRowListener(long, GeometrySimplifier, GeoJsonMultiLineWriter, int, Predicate, long, GeometryFactory, int, double)}
+   * @deprecated Use {@link BaseRowListener#BaseRowListener(long, long, GeometrySimplifier, GeoJsonMultiLineWriter, int, Predicate, long, GeometryFactory, int, double)}
    *
    */
   @Deprecated
   public BaseRowListener(
+      long NmSplit,
```

Comment:
We need to support backwards compatibility.  We should create a new class BaseRowListenerConfiguration which is a POJO that contains all the constructor parameters here.  It would be nice if this was an immutible object constructed with a builder pattern.   

The arguments for the two current constructors should be put back to the way they were and marked as @Deprecated.  A new constructor should be added that takes a BaseRowListenerConfiguration object.  In the future, new configuration parameters can be added to BaseRowListenerConfiguration class and backwards compatibility will be maintained.